app/src/main/java/com/sopt/clody/presentation/utils/extension/TimeZoneExt.kt

```
@@ -0,0 +1,56 @@
+        TimePeriod.AM -> if (hour == "12") 0 else hour.toInt()
+        TimePeriod.PM -> if (hour == "12") 12 else (hour + 12).toInt()
+    }
+
+    val userTime = LocalTime.of(hour24, minute.toInt())
+    val userZoned = ZonedDateTime.of(LocalDate.now(userZoneId), userTime, userZoneId)
+    val kstZoned = userZoned.withZoneSameInstant(kstZoneId)
+
+    val kstHour = kstZoned.hour
+    val kstMinute = kstZoned.minute
+
+    return String.format("%02s:%02s", kstHour, kstMinute)
```

detekt (1.23.8)

[warning] String.format("%02s:%02s", kstHour, kstMinute) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)